### PR TITLE
Misc AmCharts Fixes

### PR DESCRIPTION
**Pull request by mlzummo**
AmCharts DataProvider updated on related PP changes & columns()
Color By Category Fixed

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @dtsnell4 Please Review

**Comment by mlzummo** [comment]
needed to access paletteGrouping by using __prop_ that ok?

**Comment by mlzummo** [comment]
@GordonSmith if this is good it needs to go into 1.8.2

File: src/amchart/CommonSerial.js [hunk 1]
```diff
@@ -376,10 +378,21 @@
...
             this._chart.dataProvider = this.amFormatData(this.data());
         }
+        this._chart.dataProvider = this.amFormatData(this.data());
         this._lastDataUpdated = this._dataUpdated;
+        this._lastColumnsUpdated = this._columnsUpdated;
+        this._lastYAxisType = this.yAxisType();
+        this._lastXAxisType = this.xAxisType();
+        this._lastXAxisTypeTimePattern = this.xAxisTypeTimePattern();
+        this._lastYAxisTypeTimePattern = this.yAxisTypeTimePattern();
+        this._lastPaletteGrouping =  this.__prop_paletteGrouping;
```
Comment (GordonSmith) [inline src/amchart/CommonSerial.js]: by convention we use "_prevXXX"

**Comment by GordonSmith** [comment]
If you switch the _last to _prev I will pull.

**Comment by mlzummo** [comment]
becuase paletteGrouping is defined in columns.js not commonserial.js and u cant access it from both like that ?

**Pull request merged**